CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.7] - 2020-04-08
+
+### Added
+
+- Add new parameter `namespace` to `push-to-app-collection` job which allows
+  configuring namespace where chart should be installed.
```

Comment:
For the future: that part should be included in the previous PR. Only the header and links at the bottom should go here.